python/packages/ai/teams/ai/actions/action_entry.py

```diff
@@ -26,3 +26,8 @@ def __init__(
         self.name = name
         self.allow_overrides = allow_overrides
         self.func = func
+
+    async def invoke(
+        self, context: TurnContext, state: StateT, entities: Any, name: Optional[str] = None
```

Comment:
Is it possible to be more specific than `Any` here?

Reply:
the entities arg can be anything truly since its user defined, we could switch to `Dict[str, Any]` but thats essentially the same thing.

Reply:
I'd prefer the later since that's how we do it in `js` but I'll defer to your experience with python

Reply:
I tried but for python that results in type errors since we what to pass class instances in some cases 
![image](https://github.com/microsoft/teams-ai/assets/44811138/5bbbf3bc-d3d7-4b06-9d42-d6203ee0d4d9)